tests/test_shape_families.py

```diff
@@ -0,0 +1,93 @@
+from coxeter.shape_families import (family_from_doi, Family323Plus, Family423,
+                                    Family523, RegularNGonFamily,
+                                    TruncatedTetrahedronFamily)
+import numpy as np
+import pytest
+
+
+@pytest.mark.parametrize('n', range(3, 100))
+def test_regular_ngon(n):
+    family = RegularNGonFamily()
+    poly = family(n)
+    assert len(poly.vertices) == n
+    # All side lengths should be the same.
+    assert len(np.unique(np.linalg.norm(
+        poly.vertices - np.roll(poly.vertices, shift=-1, axis=0),
+        axis=1).round(4))) == 1
+
+
+def test_shape_repos():
+    family = family_from_doi('10.1126/science.1220869')[0]
+    found_cube = False
+    for key, shape_data in family.data.items():
+        if shape_data['name'] == 'Cube':
+            found_cube = True
+            break
+    if not found_cube:
+        assert False, "Could not find a cube in the dataset."
```

Comment:
Ideal case for a `for...else`.
```suggestion
    for key, shape_data in family.data.items():
        if shape_data['name'] == 'Cube':
            break
    else:
        assert False, "Could not find a cube in the dataset."
```

Reply:
Good call, I always forget this construct exists.